src/index.ts

```diff
@@ -79,41 +83,35 @@ export function usePagination({
     );
 
     return {
-        setPage: React.useCallback(
-            (page: number) => {
-                dispatch({
-                    type: "SET",
-                    page,
-                });
-            },
-            [dispatch]
-        ),
+        setPage: React.useCallback((page: number) => {
+            dispatch({
+                type: "SET",
+                page,
+            });
+        }, []),
```

Comment:
It's true that the reference to `dispatch` should never change, but doesn't React team recommend passing these things as dependencies anyways as a good practice?

Reply:
🤷  not that I've seen. [From the react docs](https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often):

```js
function Counter() {
  const [count, setCount] = useState(0);

  useEffect(() => {
    const id = setInterval(() => {
      setCount(c => c + 1); // ✅ This doesn't depend on `count` variable outside
    }, 1000);
    return () => clearInterval(id);
  }, []); // ✅ Our effect doesn't use any variables in the component scope

  return <h1>{count}</h1>;
}
```

Happy to add them back, LMK!